### DISCUSSION TITLE
Is there a way to know what is the default gitlab server the cli is working against using the cli (without inspecting the config file)?

### DISCUSSION THREAD

**Discussion by w7089** [python-gitlab/python-gitlab#2526] [Q&A]
I wonder if there is a way to know what is the default gitlab server the cli is working against using the cli command (without inspecting the config file `~/.python-gitlab.cfg`)?

**Comment by nejch** [comment]
@w7089 yes, currently the easiest way I can think of is supplying the `--debug` flag which will print the HTTP logs as well.

```shell
$ gitlab --debug project list
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): gitlab.com:443
...
```

Is this what you had in mind?

**Reply by w7089** [reply]
Yes, it works. But I thought about some cli command for this :) So it would show the default gitlab server specified in config file

**Comment by w7089** [comment]
Found what i'm looking for at `glab` cli. Thought it's possible in python gitlab:

```
glab config get host
```

**Reply by nejch** [reply]
If it's something you'd like to see in python-gitlab feel free to open an issue so someone can work on it if they decide to.

Or not sure if this was just confusion between `glab` and `gitlab` .. :)
